### ISSUE TITLE
Users created/logged in with 1.5.0 cannot access the app after upgrading to 2.0.0-pre*

### ISSUE THREAD

**Issue by astorije** [thelounge/lounge#569]
Steps to reproduce:

- On The Lounge v1.5.0, create a user
- Start the server, login and close the server
- Install the latest version (v2.0.0-pre6 right now)
- Start the server and notice the following error in the server console:

```js
Missing error handler on `socket`.
TypeError: Cannot read property 'token' of undefined
    at Socket.auth (/Users/jeremie/Perso/lounge/src/server.js:289:60)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Socket.onevent (/Users/jeremie/Perso/lounge/node_modules/socket.io/lib/socket.js:335:8)
    at Socket.onpacket (/Users/jeremie/Perso/lounge/node_modules/socket.io/lib/socket.js:295:12)
    at Client.ondecoded (/Users/jeremie/Perso/lounge/node_modules/socket.io/lib/client.js:193:14)
    at Decoder.Emitter.emit (/Users/jeremie/Perso/lounge/node_modules/socket.io/node_modules/socket.io-parser/node_modules/component-emitter/index.js:134:20)
    at Decoder.add (/Users/jeremie/Perso/lounge/node_modules/socket.io/node_modules/socket.io-parser/index.js:247:12)
    at Client.ondata (/Users/jeremie/Perso/lounge/node_modules/socket.io/lib/client.js:175:18)
    at emitOne (events.js:77:13)
```

**Comment by PugaBear** [comment]
I first experienced this problem after upgrading to pre6. After downgrading back to the commit I had last updated with (the addition of custom highlights), I still cannot load the page.

astorije pointed out that it does work if you use incognito/private browsing, so it must be some issue with local data. I hope a fix for this can be found while leaving the local data intact, because I had a user who spent many days working on custom css.

**Comment by PugaBear** [comment]
I can't figure out why but I am able to log in while none of my other users are. I can view all of my custom css and highlights.

**Comment by toxinu** [comment]
Same bug for me.
I can't log. Any workaround?

**Comment by astorije** [comment]
@toxinu, there is a bug fix under review: #570.
In the meantime, you can either log in with a different browser, or in private browsing, or by emptying your browser cache (but careful, that **will** delete your custom CSS if you have some, as well as settings).

**Comment by toxinu** [comment]
@astorije, browser cache deleted. Thank you! :+1:
